src/Packetery/Module/Options/Repository.php

```diff
@@ -67,4 +58,49 @@ public function deleteExpiredTransientsByPrefix( string $prefix ) {
 			)
 		);
 	}
+
+	/**
+	 * @param array<string> $prefixes
+	 *
+	 * @return string[]
+	 */
+	public function getAllTransientsByPrefixes( array $prefixes ): array {
+		if ( [] === $prefixes ) {
+			return [];
+		}
+
+		$optionPrefixes = [];
+		foreach ( $prefixes as $prefix ) {
+			$optionPrefixes[] = '_transient_' . $prefix;
+		}
+
+		$transientNames = [];
+		$result         = $this->getAllOptionNamesByPrefixes( $optionPrefixes );
+
+		foreach ( $result as $optionName ) {
+			$transientNames[] = preg_replace( '~^_transient_~', '', $optionName );
+		}
+
+		return $transientNames;
+	}
+
+	/**
+	 * @param array<string> $prefixes
+	 *
+	 * @return string[]
+	 */
+	public function getAllOptionNamesByPrefixes( array $prefixes ): array {
```

Comment:
Jednotlivé options bych netahal přímo z DB, ale využil bych metodu WP wp_load_alloptions() - https://developer.wordpress.org/reference/functions/wp_load_alloptions/.   Pokud by WP změnil způsob načítání options, museli bychom dělat změny.  Použití funkce WP je tak bezpečnější.

Reply:
moc jsem tuto funkci nezkoumal, pokud není dostačující, zkus se podívat, jestli najdeš jinou, vhodnější.